### PR TITLE
Fix media manager 'up' button

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #16860

### Summary of Changes

This PR fixes the 403 error that's generated when trying to move back a directory in the Media Manager when logged in as a Publisher

### Testing Instructions

- Create a new user and set the user group to Publisher
- In the frontend, go to edit an article and in Tinymce, click the "Image" button
- Navigate to a directory
- Click the "Up" button to go back to the original directory

### Expected result

You get navigated back to the directory as expected

### Actual result

403 error. Page cannot be found

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e6186c5c003b3ca432d21b9d2736097534e1c587<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19443">issues.joomla.org/tracker/joomla-cms/19443</a>.</sub>

**Comment by anuragteapot** [comment]
I have tested this item :white_check_mark: successfully on e6186c5c003b3ca432d21b9d2736097534e1c587<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19443">issues.joomla.org/tracker/joomla-cms/19443</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

File: media/media/js/popup-imagemanager.js [hunk 1]
```diff
@@ -27,6 +27,11 @@
 			var o = this.getUriObject(window.self.location.href),
 				q = this.getQueryObject(o.query);
 
+			var options = Joomla.getOptions('mediamanager');
+
+			this.author = options.author;
+			this.base   = options.base;
```
Comment (ggppdk) [inline media/media/js/popup-imagemanager.js:33]: Should be
`this.base   = options.base || image_base_path;`
to support existing template overrides

**Comment by zero-24** [comment]
@ggppdk can you please do a PR. So we can implement that change in 3.8.5?

**Pull request merged**